### DISCUSSION TITLE
Telegram Bot Framework

### DISCUSSION THREAD

**Discussion by gersonfreire** [python-telegram-bot/python-telegram-bot#4468] [Feature Suggestions]
Hi! Could you mention by any means my new Telegram bot application Framework https://github.com/gersonfreire/telegram-bot-framework? It´s fully based on python-telegram-bot!

**Comment by Bibo-Joshi** [comment]
Hi. You're welcome to link your project on https://github.com/python-telegram-bot/python-telegram-bot/wiki/Related-Projects. Always nice to see what people built on top of PTB :)
